src/main/java/com/wwi21sebgroup5/cinema/entities/Ticket.java

```diff
@@ -20,5 +22,27 @@ public class Ticket {
     @NotNull
     @GeneratedValue(strategy = GenerationType.AUTO)
     private UUID id;
-    
+
+    @ManyToOne
+    @JoinColumn(name = "event_id", referencedColumnName = "id")
+    private Event event;
+
+    @OneToOne
+    @JoinColumn(name = "qrcode_id", referencedColumnName = "id")
+    private QR_Code qr_code;
+
+    @OneToOne
+    @JoinColumn(name = "seat_id", referencedColumnName = "id")
+    private Seat seat;
+
+    @ManyToOne
+    @JoinColumn(name = "booking_id", referencedColumnName = "id")
+    private Booking booking;
+
+    public Ticket(Event pEvent, Seat pSeat, Booking pBooking){
+        event = pEvent;
+        seat = pSeat;
+        booking = pBooking;
```

Comment:
würd das nich mit in den konstruktor aufnehmen, wird ja erst angelegt, wenn es gebucht wird